Testscripts/Linux/gpu-driver-install.sh

```diff
@@ -29,45 +28,55 @@ grid_driver="https://go.microsoft.com/fwlink/?linkid=874272"
 
 #######################################################################
 function InstallCUDADrivers() {
+    LogMsg "Starting CUDA driver installation"
     case $DISTRO in
     redhat_7|centos_7)
-        CUDA_REPO_PKG="cuda-repo-rhel7-${CUDADriverVersion}.x86_64.rpm"
-        LogMsg "Using ${CUDA_REPO_PKG}"
+        CUDA_REPO_PKG="cuda-repo-rhel7-$CUDADriverVersion.x86_64.rpm"
+        LogMsg "Using $CUDA_REPO_PKG"
 
         wget http://developer.download.nvidia.com/compute/cuda/repos/rhel7/x86_64/"${CUDA_REPO_PKG}" -O /tmp/"${CUDA_REPO_PKG}"
         if [ $? -ne 0 ]; then
-            LogErr "Failed to download ${CUDA_REPO_PKG}"
+            LogErr "Failed to download $CUDA_REPO_PKG"
             SetTestStateAborted
             return 1
+        else
+            LogMsg "Successfully downloaded the $CUDA_REPO_PKG file in /tmp directory"
         fi
 
-        rpm -ivh /tmp/"${CUDA_REPO_PKG}"
+        rpm -ivh /tmp/"$CUDA_REPO_PKG"
+        LogMsg "Installed the rpm package, $CUDA_REPO_PKG"
         yum --nogpgcheck -y install cuda-drivers > ${HOME}/install_drivers.log 2>&1
         if [ $? -ne 0 ]; then
             LogErr "Failed to install the cuda-drivers!"
             SetTestStateAborted
             return 1
+        else
+            LogMsg "Successfully installed cuda-drivers"
         fi
     ;;
 
     ubuntu*)
         GetOSVersion
         # Temporary fix till driver for ubuntu19 series list under http://developer.download.nvidia.com/compute/cuda/repos/
         if [[ $os_RELEASE =~ 19.* ]]; then
+            LogMsg "There is no cuda driver for $os_RELEASE, used the one for 18.10"
             os_RELEASE="18.10"
         fi
         CUDA_REPO_PKG="cuda-repo-ubuntu${os_RELEASE//./}_${CUDADriverVersion}_amd64.deb"
         LogMsg "Using ${CUDA_REPO_PKG}"
 
         wget http://developer.download.nvidia.com/compute/cuda/repos/ubuntu"${os_RELEASE//./}"/x86_64/"${CUDA_REPO_PKG}" -O /tmp/"${CUDA_REPO_PKG}"
         if [ $? -ne 0 ]; then
-            LogErr "Failed to download ${CUDA_REPO_PKG}"
+            LogErr "Failed to download $CUDA_REPO_PKG"
             SetTestStateAborted
             return 1
+        else
+            LogMsg "Successfully downloaded $CUDA_REPO_PKG"
         fi
 
         apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/cuda/repos/ubuntu"${os_RELEASE//./}"/x86_64/7fa2af80.pub
-        dpkg -i /tmp/"${CUDA_REPO_PKG}"
+        dpkg -i /tmp/"$CUDA_REPO_PKG"
+        LogMsg "Installed $CUDA_REPO_PKG"
```

Comment:
Should say "Installing ..."

Reply:
Again, the logging is after the command execution.